src/main/webapp/partials/instructorHelpStudents.jsp

```diff
@@ -0,0 +1,609 @@
+                        <td><span class="highlight">jack@email.com</span></td>
+                        <td class="no-print align-center">
+                          <a class="btn btn-default btn-xs" title="View details of the student"
+                             href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">View</a>
+                          <a class="btn btn-default btn-xs" title="Use this to edit the details of this student. <br>To edit multiple students in one go, you can use the enroll page: <br>Simply enroll students using the updated data and existing data will be updated accordingly"
+                             href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">Edit</a>
+                          <a class="course-student-delete-link btn btn-default btn-xs"
+                             title="Delete the student and the corresponding submissions from the course" href="" data-toggle="tooltip" data-placement="top">Delete</a>
+                          <a class="btn btn-default btn-xs" href="" title="View all data about this student" target="_blank" rel="noopener noreferrer"
+                             data-toggle="tooltip" data-placement="top">All Records</a>
+                        </td>
+                      </tr>
+                      </tbody>
+                    </table>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="panel panel-default" id="student-email">
+      <div class="panel-heading cursor-pointer" data-toggle="collapse" data-target="#student-email-body">
+        <h3 class="panel-title">How do I email a student or students in my course?</h3>
+      </div>
+      <div id="student-email-body" class="panel-collapse collapse">
+        <div class="panel-body">
+          <ol>
+            <li>
+              On the <b>Students</b> page, filter out the selected student/students.
+            </li>
+            <li>
+              Click <button class="btn btn-xs btn-default">Copy Emails</button> to copy their email addresses to your clipboard
+            </li>
+            <li>
+              Use your preferred email provider to email the students
+            </li>
+          </ol>
+        </div>
+      </div>
+    </div>
+  </div>
+  <h3>Student Accounts</h3>
+  <div class="panel-group">
+    <div class="panel panel-default" id="student-google-account">
+      <div class="panel-heading cursor-pointer" data-toggle="collapse" data-target="#student-google-account-body">
+        <h3 class="panel-title">Is it compulsory for students to use Google accounts?</h3>
+      </div>
+      <div id="student-google-account-body" class="panel-collapse collapse">
+        <div class="panel-body">
+          <p>
+            Student can submit feedback and view results without logging into TEAMMATES with their Google account.
```

Comment:
Also, I believe they will be forced to login if they have previously linked their Google account. Something like this?
`Students can submit feedback and view results without having to login to TEAMMATES, unless they choose to link their Google account (optional).`